### PR TITLE
Add Decarbonization Impacts Use Case

### PR DESCRIPTION
**Pull request by ssuffian**
I believe this is an important use case for the power systems data work, and it is not fully captured (although it is related to) the consumption optimization use case.

### CODE REVIEW

File: website/use-cases/index.markdown [hunk 1]
```diff
@@ -75,6 +75,19 @@ To make impactful clean energy siting decisions and to measure the carbon impact
 
 * Organisations with climate goals that can target their clean energy investments to grids with higher emissions.
 
+## Use Case 4: Measurement of Decarbonization Projects <a id="use-case-decarb-projects" href="#use-case-decarb-projects" class="permalink">🔗</a>
+
+Decarbonization can be accelerated through the accurate measuring of the impacts of various decarbonization projects, such as HVAC electrification and energy efficiency measures.
```
Comment (Henry-WattTime) [inline website/use-cases/index.markdown:80]:
```suggestion
Decarbonization can be accelerated through the accurate measurement of the impacts of various decarbonization projects, such as HVAC electrification and energy efficiency measures.  
```

File: website/use-cases/index.markdown [hunk 2]
```diff
@@ -75,6 +75,19 @@ To make impactful clean energy siting decisions and to measure the carbon impact
 
 * Organisations with climate goals that can target their clean energy investments to grids with higher emissions.
 
+## Use Case 4: Measurement of Decarbonization Projects <a id="use-case-decarb-projects" href="#use-case-decarb-projects" class="permalink">🔗</a>
+
+Decarbonization can be accelerated through the accurate measuring of the impacts of various decarbonization projects, such as HVAC electrification and energy efficiency measures.  
+
+### Needs
+
+To make trustworthy claims on the impact of decarbonization projects, access to accurate and detailed grid data is required to calculate the carbon impacts based on changes in energy consumption.
```
Comment (Henry-WattTime) [inline website/use-cases/index.markdown:84]:
```suggestion
To make trustworthy claims on the impact of decarbonization projects, access to accurate and detailed grid data is required to calculate the carbon impacts based on changes in electricity consumption. 
```

**Comment by Henry-WattTime** [comment]
Few minor textual tweaks, but otherwise looks like a great use case

File: website/use-cases/index.markdown [hunk 3]
```diff
@@ -75,6 +75,19 @@ To make impactful clean energy siting decisions and to measure the carbon impact
 
 * Organisations with climate goals that can target their clean energy investments to grids with higher emissions.
 
+## Use Case 4: Measurement of Decarbonization Projects <a id="use-case-decarb-projects" href="#use-case-decarb-projects" class="permalink">🔗</a>
```
Comment (halliecramer1) [inline website/use-cases/index.markdown:78]: Suggest being more specific and calling this: "Electrification and Energy Efficiency Projects". Based on how the others are framed, I think "Measurement of" is already implied and explained further below.

File: website/use-cases/index.markdown (hunk 1 above)
Comment (halliecramer1) [inline website/use-cases/index.markdown:80]: various "demand-side decarbonization" or "behind-the-meter decarbonization" - this should be distinct from the needs of the other decarbonization actions above (RE siting, load shifting).

File: website/use-cases/index.markdown (hunk 2 above)
Comment (halliecramer1) [inline website/use-cases/index.markdown:84]: "impact of these types of decarbonization projects"

File: website/use-cases/index.markdown [hunk 4]
```diff
@@ -75,6 +75,19 @@ To make impactful clean energy siting decisions and to measure the carbon impact
...
+
+### Needs
+
+To make trustworthy claims on the impact of decarbonization projects, access to accurate and detailed grid data is required to calculate the carbon impacts based on changes in energy consumption. 
+
+### Example users
+
+* Organisations conducting measurement and verification of decarbonization projects.
+
+* Organisations managing registries for carbon offsets or energy attribute certificates.
```
Comment (halliecramer1) [inline website/use-cases/index.markdown:90]: Without further explanation, this is very confusing, as the well known definition of energy attribute certificates is for energy produced, not avoided

**Review by halliecramer1 — COMMENTED** [review]
Generally supportive of this but think a couple of clarifications should be added to how it is distinct from the other use cases.

File: website/use-cases/index.markdown (hunk 4 above)
Comment (ssuffian) [reply website/use-cases/index.markdown:90]: We can change it to environmental attribute certificate if that matches the current industry definitions better.  Although the cleanest way to "produce" energy is by avoiding it!

File: website/use-cases/index.markdown (hunk 3 above)
Comment (ssuffian) [reply website/use-cases/index.markdown:78]: Sounds good. I added Storage as an option as well to cover the three primary types of projects that I can think of that aren't covered by the other use cases.

**Pull request merged**